features/zoeken/met-nummeraanduiding-identificatie/overzicht.feature

```diff
@@ -0,0 +1,59 @@
+# language: nl
+Functionaliteit: zoek personen met de nummeraanduiding identificatie van een adres
+  Als consumer
+  wil ik de personen vinden door het opgeven van de nummeraanduiding identificatie van het adres waar zij staan ingeschreven
+
+  Regel: personen ingeschreven op het adres waarmee wordt gezocht, worden gevonden
+
+    Scenario: één persoon staat ingeschreven op het adres waarmee wordt gezocht
+      Gegeven adres 'A1' in gemeente 'Rotterdam'
+      En de persoon 'Robin'
+      En 'Robin' is 3 jaar geleden ingeschreven op adres 'A1'
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met nummeraanduiding identificatie van 'A1'
+      Dan wordt 'Robin' gevonden
+
+    Scenario: meerdere personen staan ingeschreven op het adres waarmee wordt gezocht
+      Gegeven adres 'A1' in gemeente '\'s-Gravenhage'
```

Comment:
Hier zelfde opmerking als bij adresseerbaar object identificatie: Waarom wordt hier wel gemeente genoemd, waarmee wordt gevarieerd en met waarde waarbij backslash nodig is. En waarom niet nummeraanduiding identificatie, terwijl dat is waar dit scenario over gaat.